src/paint_canvas/chunk.rs

```diff
@@ -56,13 +48,12 @@ impl Chunk {
    /// Marks the chunk as dirty - that is, invalidates any cached PNG and WebP data,
    /// and marks it as unsaved.
    pub fn mark_dirty(&mut self) {
-      self.image_cache = None;
-      self.saved = false;
+      self.dirty = true;
    }
 
    /// Marks the given sub-chunk within this master chunk as saved.
    pub fn mark_saved(&mut self) {
-      self.saved = true;
+      self.dirty = false;
    }
```

Comment:
How is saving handled now? After all, we need to know whether a chunk was touched since last save, so as not to waste I/O on resaving the same chunk.

Reply:
Hmm, interesting... I could've sworn I had used it before but NetCanv is soon turning two years old, so my memory of certain things may be a bit hazy 😄